azure-batch/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+2.0.0 (2017-02-17)
+++++++++++++++++++
+
+- AAD token authentication now supported.
+- Support for running tasks under a predefined or automatic user account.
```

Comment:
Should we mention the changed method names like pool_list_pool_usage_metrics -> pool_list_usage_metrics, or is that too detailed for this history file?